qutebrowser/config/configdata.yml

```diff
@@ -291,15 +292,15 @@ content.headers.custom:
       name: String
       encoding: ascii
     none_ok: true
-  desc: Set custom headers for qutebrowser HTTP requests.
+  desc: The custom headers for qutebrowser HTTP requests.
```

Comment:
This sounds weird to me. Maybe "Add custom headers", changing this to imperative mood?

Reply:
@The-Compiler: I thought about this again and in the latest version we ended up using imperative mood for boolean settings.
That's not really the case here and I think now that I changed this to "Add custom headers" it kind of implies I can choose whether or not to add a custom header here. This is not true as really I can choose the (potentially empty) content of that header.
As we dropped the articles for most of those cases, how about changing this to "Custom headers to add for qutebrowser HTTP requests." or even simply "Custom headers for qutebrowser HTTP requests." instead?

Reply:
"Custom headers for qutebrowser HTTP requests." sounds good to me!